tests/foreman/cli/test_classparameters.py

```diff
@@ -328,15 +471,26 @@ def test_positive_validate_matcher_value_required_check(self):
         5.  Submit the change.
 
         @assert: Error not raised for matcher value.
-
-        @caseautomation: notautomated
-
-        @CaseLevel: System
         """
+        sc_param_id = self.sc_params_ids_list.pop()
+        SmartClassParameter.add_override_value({
+            'smart-class-parameter-id': sc_param_id,
+            'match': 'domain=example.com',
+            'value': gen_string('alpha')
+        })
+        SmartClassParameter.update({
+            'id': sc_param_id,
+            'override': 1,
+            'required': 1
+        })
+        sc_param = SmartClassParameter.info({
+            'puppet-class': 'ntp',
+            'id': sc_param_id,
+        })
+        self.assertEqual(sc_param['required'], True)
 
```

Comment:
assert if the matcher value is set as given. It lists all overrides in hammer 'info'.